usaspending_api/etl/management/commands/combine_universal_transaction_matview_chunks.py

```diff
@@ -0,0 +1,162 @@
+import logging
+import asyncio
+import sqlparse
+from pathlib import Path
+
+from django.db import connection, transaction
+from django.core.management.base import BaseCommand
+from usaspending_api.common.data_connectors.async_sql_query import async_run_creates
+from usaspending_api.common.helpers.timing_helpers import ConsoleTimer as Timer
+from usaspending_api.common.matview_manager import DEFAULT_CHUNKED_MATIVEW_DIR
+
+logger = logging.getLogger("script")
+
+
+class Command(BaseCommand):
+
+    help = """
+    This script combines the chunked Universal Transaction Matviews and
+    combines them into a single table.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("--chunk-count", default=10, help="Number of chunked matviews to read from", type=int)
+        parser.add_argument(
+            "--matview-dir",
+            type=Path,
+            help="Choose a non-default directory to store materialized view SQL files.",
+            default=DEFAULT_CHUNKED_MATIVEW_DIR,
+        )
+        parser.add_argument(
+            "--keep-old-data",
+            action="store_true",
+            default=False,
+            help="Indicates whether or not to drop old table at end of command",
+        )
+        parser.add_argument(
+            "--keep-matview-data",
+            action="store_true",
+            default=False,
+            help="Indicates whether or not to empty data from chunked matviews at the end of command",
```

Comment:
So just to be clear, we're going with the default of emptying the matview data out each time the table is generated, and then the refresh will refresh it from empty to full on each run of the pipeline?
(this is what `default=False` will lead to).

Reply:
That's correct @kwhickey